### PR TITLE
Add teams field to O365 org & modify credentials models

### PR DESCRIPTION
**Pull request by Juchazari**
http://doc.10.api.iland.test/1.0/#/Office%20365%20Backup/getOrganization
http://doc.10.api.iland.test/1.0/#/Office%20365%20Backup/modifyOrganizationCredentials

### CODE REVIEW

File: src/sdk/model/o365/o365-organization.ts [hunk 1]
```diff
@@ -83,6 +83,13 @@ export class O365Organization extends Entity {
     return this._json.region;
   }
 
+  /**
+   * Get whether to use modern authentication.
+   */
+  get useModernAuth(): boolean {
```
Comment (ariquinones) [inline src/sdk/model/o365/o365-organization.ts:89]: Add return docs

File: src/sdk/model/o365/o365-organization.ts [hunk 2]
```diff
@@ -104,6 +111,13 @@ export class O365Organization extends Entity {
     return this._json.last_backup_time;
   }
 
+  /**
+   * Is the O365 Organization teams online
+   */
+  get isTeamsOnline(): boolean {
```
Comment (ariquinones) [inline src/sdk/model/o365/o365-organization.ts:117]: Add return docs

File: src/sdk/model/o365/o365-organization.ts (hunk 1 above)
Comment (Juchazari) [reply src/sdk/model/o365/o365-organization.ts:96]: Updated!

File: src/sdk/model/o365/o365-organization.ts (hunk 2 above)
Comment (Juchazari) [reply src/sdk/model/o365/o365-organization.ts:128]: Updated!

**Review by iamtash — APPROVED** [review]

**Review by ariquinones — APPROVED** [review]

**Pull request merged**